wandb/internal/handler.py

```diff
@@ -44,7 +44,6 @@ def handle(self, record):
         assert record_type
         handler_str = "handle_" + record_type
         handler = getattr(self, handler_str, None)
-        logger.debug("handle: {}".format(record_type))
```

Comment:
I got rid of this log message because it was incredibly noisy with tons of "handle: output" messages.